ansible/roles/monitoring-statsd-fe-influxdb/templates/telegraf.conf.j2

```diff
@@ -115,6 +246,15 @@
       "nsq.*.topic.* app.host.measurement.topic_name.measurement*",
       "nsq.* app.host.measurement*",
       "kairos.* app.env.measurement*",
+      "mozlocal*.* app.measurement* local_db",
```

Comment:
again, all these lines could be generated by code and configuration, not statically.